# Goal Point

# Request
motion_controller_pkg/GoalCommand goal_command          # Goal command message
---
# Result
bool success                                            # Point reached?
---
# Feedback
geometry_msgs/PoseStamped current_pose                  # Current pose (m)
geometry_msgs/TwistStamped current_speed                # Current speed (m/s)
builtin_interfaces/Duration navigation_time             # Time from departure (s)
builtin_interfaces/Duration estimated_time_remaining    # Time to goal (s)
float32 distance_remaining                              # Distance to goal (m)